src/bench_internal.c

```diff
@@ -262,8 +262,12 @@ void bench_group_jacobi_var(void* arg, int iters) {
 
     for (i = 0; i < iters; i++) {
         j += secp256k1_gej_has_quad_y_var(&data->gej[0]);
+        secp256k1_fe_add(&data->gej[0].y, &data->fe[1]);
+        secp256k1_fe_add(&data->gej[0].z, &data->fe[2]);
+        secp256k1_fe_normalize_var(&data->gej[0].y);
+        secp256k1_fe_normalize_var(&data->gej[0].z);
```

Comment:
Maybe add a comment that this does not result in valid points but we don't care and that the additions and normalizations are negligible (if this agrees with your intention). The latter is also true for the other two changed functions.

Reply:
Done.